pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.google.sps</groupId>
+    <artifactId>step7-2020</artifactId>
+    <version>1</version>
+    <packaging>war</packaging>
+
+    <properties>
+        <!-- This project uses Java 8 -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.appengine</groupId>
+            <artifactId>appengine-api-1.0-sdk</artifactId>
+            <version>1.9.59</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- Provides `mvn package appengine:run` for local testing
+                 and `mvn package appengine:deploy` for deploying. -->
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>appengine-maven-plugin</artifactId>
+                <version>2.2.0</version>
+                <configuration>
+                    <deploy.projectId>INPUT_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
I'd like to find a way of not including the project ID in the code. This is not a properly of the implementation and more just an artifact of the testing environment.

Reply:
The documentation says: "The Google Cloud Platform project name to use for this invocation. You must specify a projectId or you can set GCLOUD_CONFIG to use the project set in your gcloud config state."

Once we have our GCP project, we should look into the configuration to avoid this